src/test/java/com/prgrms/amabnb/room/repository/RoomRepositoryTest.java

```diff
@@ -167,7 +167,12 @@ private Room createRoom(User host) {
             .roomOption(roomOption)
             .roomType(RoomType.APARTMENT)
             .roomScope(RoomScope.PRIVATE)
+            .roomImages(List.of(createRoomImage(), createRoomImage()))
             .host(host)
             .build();
     }
+
+    private RoomImage createRoomImage() {
+        return new RoomImage(null, "aa");
```

Comment:
```suggestion
        return new RoomImage("aa");
```
develop 브랜치에 보니 RoomImage에 id 값 없는 생성자도 있네요!

Reply:
ㅎㅎㅎㅎ 그렇네욯 ㅎㅎ